featurebyte/models/feature.py

```diff
@@ -192,22 +192,14 @@ class FeatureModel(FeatureByteCatalogBaseDocumentModel):
         When the Feature get updated
     """
 
-    dtype: DBVarType = Field(
-        allow_mutation=False, description="database variable type for the feature"
-    )
+    dtype: DBVarType = Field(allow_mutation=False, default=DBVarType.UNKNOWN)
```

Comment:
Curious in actual code path, will the dtype ever be not provided when creating a FeatureModel?